React/src/DataCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+function ResourceCell(props: any): JSX.Element {
+  function isWeekEnd(date: Date): boolean {
```

Comment:
Question to @andrewmakarov 

Should we use here ```useCallback``` or it will be overkill?
In ```App.tsx``` we use it in a similar case, so in any way let's use (or not) ```useCallback``` in all cases like this :)

Reply:
Better to move this util func outside of component, seems it doesn't use any component state

Reply:
Removed outside component